docs/private_shard/architecture.md

```diff
@@ -12,31 +13,35 @@ Due to the fact that by default the network’s token doesn’t have any monetar
 
 ## Proof of Authority Shard
+In a proof of authority (PoA) shard, the validation of transactions and blocks is performed by a set of pre-approved validators or nodes, rather than through a competitive mining process. Validators are identified by their reputation, identity, or stake in the network, and are responsible for verifying the authenticity and validity of transactions. PoA shards are often used in private or permissioned blockchain networks, where the validators are known and trusted entities, such as consortium members or enterprise partners. PoA shards offer faster transaction speeds and lower energy consumption compared to proof of work (PoW) or proof of stake (PoS) consensus mechanisms, but may be more centralized and vulnerable to collusion or attacks by the validators.
 
 ## Proof of Stake Shard
+In a proof of stake (PoS) shard, validators are chosen based on their stake in the network, rather than their computational power or identity. Validators are required to lock up a certain amount of cryptocurrency or token as collateral, which incentivizes them to act honestly and secure the network. In a PoS shard, validators take turns proposing and validating blocks, and are rewarded with transaction fees or newly minted tokens. PoS shards are designed to be more energy-efficient, secure, and decentralized than PoW shards, as they do not require large amounts of computational power and electricity to perform the validation process. PoS shards are often used in public blockchain networks, such as Ethereum 2.0.
 
 ## Migration from the shard from PoA to PoS
+Migrating a shard from PoA to PoS can be a complex and challenging process, as it involves changing the underlying consensus mechanism and modifying the validation and governance rules of the network. The migration process typically involves coordinating with the existing validators and stakeholders, establishing a new set of rules and incentives for PoS validation, and ensuring that the migration does not cause any disruption or loss of data or assets. The migration process may also require the development and deployment of new smart contracts, protocols, and tools to support the PoS consensus mechanism. Overall, the migration from PoA to PoS is a significant step towards achieving a more secure, scalable, and sustainable blockchain network.
```

Comment:
I would remove the part: "Overall, the migration from PoA to PoS is a significant step towards achieving a more secure, scalable, and sustainable blockchain network.".

Reply:
Migrating from PoA to PoS will not cause any disruption or loss of data or assets, it is just a matter of owners of the network agreeing to differently allocate the stake inside the network which will not be uniformly distributed like in the PoA network.